generics/helpers/solutionAndProjectTemplateUtils.js

```
@@ -10,6 +10,8 @@
  * circular dependencies when placed in the solution module.
  */
 
+const { result } = require('lodash')
```

ESLint

[error] 'result' is assigned a value but never used.

(no-unused-vars)